TACC/Core-Portal
# task/FP-1460: Resolve dependabot alerts

## Overview: ##

Resolve client and server side package vulnerabilities.

Of note:

- Bumps python from 3.6.9 to 3.7.12
- Bumps agavepy to 1.0.0a12
- Bumps pytas to 1.6.0
- Bumps celery from 4.4.7 to 5.2.2

Up in dev.cep: https://dev.cep.tacc.utexas.edu/

## Related Jira tickets: ##

* [FP-1460](https://jira.tacc.utexas.edu/browse/FP-1460)

## Testing Steps: ##
1. `make build` and run locally `make start`
2. `npm ci` and run locally `npm run dev`
3. Click around, submit jobs, etc and make sure everything works fine
